apps/dashboard/templates/meinberlin_dashboard/includes/project_list_item.html

```diff
@@ -53,12 +53,22 @@ <h3 class="list-item__title">
         <span class="list-item__action">
             <a href="{% url 'dashboard-project-moderators' slug=project.slug %}"
                class="button button--light button--small">
-                <i class="fa fa-user" aria-hidden="true"></i>
+                <i class="fa fa-user-md" aria-hidden="true"></i>
                 {% trans 'Moderators' %}
             </a>
         </span>
         {% endif %}
 
+        {% if not project.is_public %}
```

Comment:
you would have to check back with @damusp and @Janaba but i'd propose to do it separately in the first iterator and redesign it with the new dashboard